src/components/AbortionButton/AbortionButton.mdx

```diff
@@ -0,0 +1,25 @@
+import {ArgTypes, Canvas, Meta, Story} from '@storybook/blocks';
+
+import * as AbortionButtonStories from './AbortionButton.stories';
+
+<Meta of={AbortionButtonStories} />
+
+# Abortion Button
+
+The abortion button can be used to abort a submission whether the user is authenticated or not.
+
+- If the user is authenticated, then the button will have the text 'Log out' to comply with Logius
+  directives that whenever a user is logged in with DigiD a 'Log out' option must be available.
+- If the analytics tool 'GovMetric' is enabled, aborting the submission will open a new window where
+  the user can give feedback about why they decided to not fill the form until the end.
```

Comment:
if logout and govmetric is enabled -> probably also makes sense to send them to the feedback form

Reply:
This is indeed the case. I will reword so that it's clear